runtime/queries/toml/highlights.scm

```diff
@@ -1,17 +1,17 @@
 ; Properties
 ;-----------
 
-(bare_key) @property
+(bare_key) @variable.other.member
 (quoted_key) @string
 
 ; Literals
 ;---------
 
-(boolean) @constant.builtin
+(boolean) @constant.builtin.boolean
 (comment) @comment
 (string) @string
-(integer) @number
-(float) @number
+(integer) @constant.numeric.integer
+(float) @constant.numeric.float
 (offset_date_time) @string.special
 (local_date_time) @string.special
```

Comment:
We don't have a `@string.special` scope. I'm thinking this PR can become a general pass over all queries and scopes as well.

Reply:
NVM we do.